tools/Utilities/src/ViewModels/UtilitiesMainPageViewModel.cs

```diff
@@ -54,7 +54,7 @@ public UtilitiesMainPageViewModel(IExperimentationService experimentationService
             {
                 Title = stringResource.GetLocalized("ProjectIronsidesTitle"),
                 Description = stringResource.GetLocalized("ProjectIronsidesDesc"),
-                NavigateUri = "https://aka.ms/projectironsides",
+                NavigateUri = "https://github.com/microsoft/devhome/issues/2796",

```

Comment:
Did I remember Kayla saying we were supposed to use go.microsoft.com links?

Reply:
Yes, and the fwlink I created is indeed a go.microsoft.com link - but right now it links to a page that doesn't exist yet. This PR points to the same URL as in Project Ironsides' About page. I opened a new issue for a second update to use the fwlink when that page is available.

Reply:
How about we update aka.ms/projectironsides to point to https://github.com/microsoft/devhome/issues/2796. And when we have the new page, we can just update the aka.ms/projectironsides link?

Reply:
Oh that's much more sensible, thanks! Pushed a new change to use the fwlink. I'll use the other issue to track fixing the fwlink itself when we get the page.